src/engraving/dom/score.cpp

```diff
@@ -579,15 +579,16 @@ void Score::rebuildTempoAndTimeSigMaps(Measure* measure, std::optional<BeatsPerS
                     }
                 }
             }
-            if (!RealIsNull(stretch) && !RealIsNull(stretch)) {
+
+            if (!RealIsNull(stretch) && !RealIsEqual(stretch, 1.0)) {
                 BeatsPerSecond otempo = tempomap()->tempo(segment.tick().ticks());
                 BeatsPerSecond ntempo = otempo.val / stretch;
                 tempomap()->setTempo(segment.tick().ticks(), ntempo);
 
                 Fraction currentSegmentEndTick;
 
-                if (segment.next1()) {
-                    currentSegmentEndTick = segment.next1()->tick();
+                if (const Segment* nextActiveSegment = segment.nextActive()) {
+                    currentSegmentEndTick = nextActiveSegment->tick();
```

Comment:
Ideally, the slowing down of the fermata should be based on the duration of the ChordRest that the fermata is applied to (fermatas can be voice-specific, so in this case the voice-1 chord). That's often not the same as the duration of the segment itself. This kind of situation is quite common: here the fermata should last 4/4, even if the segment ticks is 1/8.
<img width="473" alt="image" src="https://github.com/user-attachments/assets/cd61ecd0-fa21-4dfa-a075-c4199946411a">
We can do this later though. For now this looks ok to me